scripts/coordinate.js

```diff
@@ -363,6 +363,12 @@ FriendlyChat.prototype.getSession = function() {
         let renderWhenInformation = data.val().whenDescription;
         let renderWhereInformation = data.val().whereDescription;
         let renderWhoInformation = data.val().whoDescription; // To-Do: Update with first names dynamically.
+        let renderWhoAmIInformation = data.val().whoAmIDescription;
+        //for DoWhops created before Who Am I was added to create field
+        //do an undefined check so it doesn't render an undefined paragraph
+        if (renderWhoAmIInformation === undefined) {
```

Comment:
I was getting undefined in testing. This seems to be working. Can we look at this together in the morning?